src/components/HeroCard/HeroCard.js

```diff
@@ -4,21 +4,40 @@ import Column from '../Column';
 import HeroIcon from '../HeroIcon';
 import CounterCard from '../CounterCard';
 import CounterTabs from '../CounterTabs';
-import HeroData from "../../utils/heroes.json";
+
+
+
+const getCounter = (heroData, counter) => {
```

Comment:
Could this just be:

`return heroData[counter.id];`?

Reply:
I don't believe so. 

When we pull in the heroData, they come in as strings since JSON format requires it. counter.id is a number, which is why I have used the for loop and type cast to a number.